contracts/libraries/Helper.sol

```diff
@@ -1,5 +1,9 @@
-// helper methods for interacting with ERC20 tokens and sending ETH that do not consistently return true/false
-library TransferHelper {
+// Helper methods for interacting with ERC20 tokens and sending ETH that do not consistently return true/false
+library Helper {
+  function ETH() internal view returns(address) {
+    return 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
```

Comment:
You could define this in a `BaseContract` and drive other contract from `BaseContract`

Reply:
We moved it back into local variables for each contract that relies on it (to save some gas).
```
 uint public constant ETH = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
 ```

Not enough to form a BaseContract just yet 

🙏